template/template.go

```diff
@@ -205,24 +205,25 @@ func (kv KV) Values() []string {
 // as this will confuse them and prevent simple things like
 // simple equality checks to fail. Map everything to float64/string.
 type Data struct {
```

Comment:
Honestly, we are worrying about 1000x scale way to often for Alertmanager at this point.
And if we reach that point, 200 more bytes of JSON will certainly not be the bottleneck.